AnkiDroid/src/main/java/com/ichi2/libanki/Collection.java

```diff
@@ -1504,7 +1504,7 @@ public CheckDatabaseResult fixIntegrity(CollectionTask.ProgressCallback progress
         File file = new File(mPath);
         CheckDatabaseResult result = new CheckDatabaseResult(file.length());
         final int[] currentTask = {1};
-        int totalTasks = (mModels.all().size() * 4) + 27; // a few fixes are in all-models loops, the rest are one-offs
+        int totalTasks = (getModels().all().size() * 4) + 25; // a few fixes are in all-models loops, the rest are one-offs
```

Comment:
I really dislike that when I originally did progress reporting on database check in #5177 I hard coded the check count, because it leads to errors like this where we fail to maintain a magic constant that you can only derive by counting the verifications we perform.

Can you revert the 27 -> 25 number?

```suggestion
        int totalTasks = (getModels().all().size() * 4) + 27; // a few fixes are in all-models loops, the rest are one-offs
```

Reply:
I can. I actually have no idea how I ever made this change. I've no idea what this is supposed to be

Reply:
Done